perf-tests/src/perf-tests.py

```diff
@@ -65,6 +66,16 @@ def check_system(core_api, jobs_api):
     else:
         sys.exit(1)
 
+    print("Checking: connection to the S3")
+    # try to connect to AWS S3
+    s3.connect()
+    buckets = s3.read_all_buckets()
+    if buckets is not None:
```

Comment:
Not sure with this condition, probably there should be thrown some exception in such cases - but it should be reported, so I'm fine with merging.